app/src/ui/banners/render-banner.tsx

```diff
@@ -65,6 +66,15 @@ export function renderBanner(
           key={'branch-already-up-to-date'}
         ></BranchAlreadyUpToDate>
       )
+    case BannerType.SuccessfulCherryPick:
+      return (
+        <SuccessfulCherryPick
+          key="successful-cherry-pick"
+          targetBranchName={banner.targetBranchName}
+          countCherryPicked={banner.countCherryPicked}
+          onDismissed={onDismissed}
+        ></SuccessfulCherryPick>
```

Comment:
Nit:
```suggestion
        />
```

I know it's not your change, but we can probably make the same small change to the `case` above with `BranchAlreadyUpToDate`.

Reply:
Sure